bootstrap/fabric/src/main/java/org/geysermc/geyser/platform/fabric/GeyserFabricDumpInfo.java

```diff
@@ -43,21 +43,27 @@
 @Getter
 public class GeyserFabricDumpInfo extends BootstrapDumpInfo {
 
-    private String platformVersion = null;
+    private final String platformName;
```

Comment:
Not sure about having `platformName` although not against it, maybe back to the `platformType` mentioned in the dump

Reply:
this addition makes sense to me - this exposes the actual server implementation name, whereas the `platformName` in `PlatformType` is just the generic platform name that Geyser targets

Reply:
Not to mention Geyser-Spigot also has a `platformName` similar to this